Cargo.toml

```diff
@@ -65,6 +65,12 @@ kona-protocol = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1
 kona-registry = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1.0-beta.13", default-features = false }
 kona-genesis = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1.0-beta.13", default-features = false }
 
+# hana
```

Comment:
TODO. Use celestiaorg/hana after https://github.com/celestiaorg/hana/pull/2 is merged.